Data/pokemon.txt

```diff
@@ -1,493 +1,493 @@
-354 Banette
-355 Duskull
-356 Dusclops
-357 Tropius
-358 Chimecho
-359 Absol
-360 Wynaut
-361 Snorunt
-362 Glalie
-363 Spheal
-364 Sealeo
-365 Walrein
-366 Clamperl
-367 Huntail
-368 Gorebyss
-369 Relicanth
-370 Luvdisc
-371 Bagon
-372 Shelgon
-373 Salamence
-374 Beldum
-375 Metang
-376 Metagross
-377 Regirock
-378 Regice
-379 Registeel
-380 Latias
-381 Latios
-382 Kyogre
-383 Groudon
-384 Rayquaza
-385 Jirachi
-386 Deoxys
-387 Turtwig
-388 Grotle
-389 Torterra
-390 Chimchar
-391 Monferno
-392 Infernape
-393 Piplup
-394 Prinplup
-395 Empoleon
-396 Starly
-397 Staravia
-398 Staraptor
-399 Bidoof
-400 Bibarel
-401 Kricketot
-402 Kricketune
-403 Shinx
-404 Luxio
-405 Luxray
-406 Budew
-407 Roserade
-408 Cranidos
-409 Rampardos
-410 Shieldon
-411 Bastiodon
-412 Burmy
-413 Wormadam
-414 Mothim
-415 Combee
-416 Vespiquen
-417 Pachirisu
-418 Buizel
-419 Floatzel
-420 Cherubi
-421 Cherrim
-422 Shellos
-423 Gastrodon
-424 Ambipom
-425 Drifloon
-426 Drifblim
-427 Buneary
-428 Lopunny
-429 Mismagius
-430 Honchkrow
-431 Glameow
-432 Purugly
-433 Chingling
-434 Stunky
-435 Skuntank
-436 Bronzor
-437 Bronzong
-438 Bonsly
-439 Mime Jr.
-440 Happiny
-441 Chatot
-442 Spiritomb
-443 Gible
-444 Gabite
-445 Garchomp
-446 Munchlax
-447 Riolu
-448 Lucario
-449 Hippopotas
-450 Hippowdon
-451 Skorupi
-452 Drapion
-453 Croagunk
-454 Toxicroak
-455 Carnivine
-456 Finneon
-457 Lumineon
-458 Mantyke
-459 Snover
-460 Abomasnow
-461 Weavile
-462 Magnezone
-463 Lickilicky
-464 Rhyperior
-465 Tangrowth
-466 Electivire
-467 Magmortar
-468 Togekiss
-469 Yanmega
-470 Leafeon
-471 Glaceon
-472 Gliscor
-473 Mamoswine
-474 Porygon-Z
-475 Gallade
-476 Probopass
-477 Dusknoir
-478 Froslass
-479 Rotom
-480 Uxie
-481 Mesprit
-482 Azelf
-483 Dialga
-484 Palkia
-485 Heatran
-486 Regigigas
-487 Giratina
-488 Cresselia
-489 Phione
-490 Manaphy
-491 Darkrai
-492 Shaymin
-493 Arceus
+bulbasaur   0.61604    grass     poison
+ivysaur     0.462721   grass     poison
+venusaur    0.5451     grass     poison
+charmander  0.724984   fire
+charmeleon  0.491232   fire
+charizard   0.601881   fire      flying
+squirtle    0.708672   water
+wartortle   0.748101   water
+blastoise   0.64938    water
+caterpie    0.718227   bug
+metapod     0.621901   bug
+butterfree  0.630765   bug       flying
+weedle      0.638418   bug       poison
+kakuna      0.773358   bug       poison
+beedrill    0.802686   bug       poison
+pidgey      0.460133   normal    flying
+pidgeotto   0.906494   normal    flying
+pidgeot     0.887968   normal    flying
+rattata     0.478938   normal
+raticate    0.640308   normal
+spearow     0.459023   normal    flying
+fearow      0.488446   normal    flying
+ekans       0.463613   poison
+arbok       0.589331   poison
+pikachu     0.851867   electric
+raichu      0.751316   electric
+sandshrew   0.77978    ground
+sandslash   0.730616   ground
+nidoran     0.759855   poison
+nidorina    0.681913   poison
+nidoqueen   0.633158   poison    ground
+nidoran     0.414212   poison
```

Comment:
Line 29 and line 32 are duplicate names. This may cause len(pokemon_dict) to be one less than len(pokemon_list).